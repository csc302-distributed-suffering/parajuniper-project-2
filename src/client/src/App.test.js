import React from 'react';
import renderer from 'react-test-renderer';
import 'regenerator-runtime/runtime'

describe('Test App Entry point', function () {
  // it('renders correctly', function () {
  // const wrapper = shallow(<App/>);
  // // expect(wrapper).toMatchSnapshot();
  // // expect(toJson(wrapper)).toMatchSnapshot();
  // });
  // it('renders correctly', () => {
  //     const tree = renderer
  //       .create(<App/>)
  //       .toJSON();
  //     expect(tree).toMatchSnapshot();
  //   });
  it('temp', () => {
    expect(true).toBe(true)
  })
});
