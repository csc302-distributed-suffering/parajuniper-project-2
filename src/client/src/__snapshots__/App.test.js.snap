// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App Entry point renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <p>
      Why, if it isn't a running React app! A truly fine sight indeed.
    </p>
    <button
      onClick={[Function]}
    >
      Click Me, I dare you! ;)
    </button>
    <div
      id="hiddenDiv"
    />
  </header>
</div>
`;
